test/router-view.spec.js

```diff
@@ -191,6 +237,11 @@ function withNamedViewport(routeConfig) {
   return component;
 }
 
+/**
+ * @param {StageComponent} component
+ * @param {RouteConfig} defaultRoute
```

Comment:
Misnamed?

Reply:
You meant `defaultRoute` is not a `RouteConfig` ? from the look of `routeConfig` in the function body, I think it's the right type.

Reply:
Woops I think I was just perusing and mis-interpretted this, you are correct, disregard.

Reply:
He's not entirely wrong. Ironically, it is `routeConfig` that is not a proper `RouteConfig`. `withNamedViewport` is consistently being passed an object with just a `moduleId` property, which is then piped to the `routeConfig` property on `configure`. I'm focusing on Binh's work so I'm not sure how this affects the other tests.

https://github.com/aurelia/router/blob/master/src/interfaces.js